pynblint/nb_linting.py

```diff
@@ -384,6 +398,14 @@ def cells_too_long(notebook: Notebook) -> List[Cell]:
         linting_function=empty_cells,
         show_details=False,
     ),
+    LintDefinition(
+        slug="multiline_comment_detected",
```

Comment:
```suggestion
        slug="long_multiline_python_comment",
```

Same as before. I suggest changing this slug for better consistency with the other slugs.